src/EditorFeatures/Core.Wpf/Suggestions/SuggestedActionsSource.cs

```diff
@@ -234,12 +234,13 @@ private void OnTextViewClosed(object sender, EventArgs e)
                     return null;
 
                 var lowPriorityAnalyzers = new ConcurrentSet<DiagnosticAnalyzer>();
+                var lowPriorityAnalyzerSupportedDiagnosticIds = new ConcurrentSet<string>();
```

Comment:
intentional that this is not read?

Reply:
It's the same as lowPriorityAnalyzers. Read/Writes on both happen inside each of the SuggestedActionPriorityProviders.